### DISCUSSION TITLE
What's up with the Atom & Electron slack?

### DISCUSSION THREAD

**Discussion by cstuartroe** [atom/atom#22869] [Q&A]
The contributing guide provides a link to an Atom & Electron slack team, but it just takes me to some Heroku landing page. It seems like either that link should be fixed or taken out of the guide. Does the slack team still exist?

**Comment by the-j0k3r** [comment]
The slack channel and all links are now dead, These discussions boards are the replacement for that.

Links need to be updated i.e. redirected here so that would be I guess a nice way to make some contribution to the project.

Here is something for context https://github.com/atom/atom/pull/22075

**Reply by sadick254** [reply]
@cstuartroe The slack team doesn't exist and also the discuss forum also doesn't exist. @the-j0k3r is right we have replaced those with Github Discussions. It is better to consolidate discussions around Atom on one platform where everyone can participate. We have updated atom.io and atom repo to reflect the change.
